cmd/juju/storage/remove.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package storage
+
+import (
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/juju/common"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+// NewRemoveStorageCommandWithAPI returns a command
+// used to remove storage from the model.
+func NewRemoveStorageCommandWithAPI() cmd.Command {
+	cmd := &removeStorageCommand{}
+	cmd.newEntityDestroyerCloser = func() (EntityDestroyerCloser, error) {
+		return cmd.NewStorageAPI()
+	}
+	return modelcmd.Wrap(cmd)
+}
+
+// NewRemoveStorageCommand returns a command
+// used to remove storage from the model.
+func NewRemoveStorageCommand(new NewEntityDestroyerCloserFunc) cmd.Command {
+	cmd := &removeStorageCommand{}
+	cmd.newEntityDestroyerCloser = new
+	return modelcmd.Wrap(cmd)
+}
+
+const (
+	removeStorageCommandDoc = `
+Removes storage (volumes/filesystems) from the model. Specify one or more
+unit storage, volume, or filesystem IDs, as output by "juju storage".
+
+Examples:
+    juju remove-storage volume-0
+    juju remove-storage filesystem-1/2
+    juju remove-storage pgdata/0
+`
```

Comment:
I presume this section is defining the help text. If so, I would like to volunteer to massage and expand it, once [this docs PR](https://github.com/juju/docs/pull/1688) is finalized.